### PR TITLE
Bundles starter pack

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| Versions      | 1.7, 1.13, 2.1, master

Bundles starter pack - added new doc

### CODE REVIEW

File: docs/getting_started/bundles_starter_pack.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Bundles starter pack
+
+eZ Platform application is a collection of bundles, similar to Symfony where ["everything is a bundle"](http://symfony.com/doc/current/book/bundles.html). Bundles listed below are useful additions to your clean eZ installation by using them you can improve your Platform usage.
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: `...additions to your clean eZ Platform installation. By using them...`

File: docs/getting_started/bundles_starter_pack.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+For more information about eZ bundle structure checkout:
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: `eZ Platform's bundle...` (let's avoid using `eZ` by itself unless referring to the company specifically).
`check out` - two words

File: docs/getting_started/bundles_starter_pack.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## Everyday Platform usage
+
+|Bundle|Description|
+|------|-----------|
+|[ezmigrationbundle](https://github.com/kaliop-uk/ezmigrationbundle)|helps to handle content upgrades/migrations|
+|[EzCoreExtraBundle](https://github.com/lolautruche/EzCoreExtraBundle)|adds: configurable template variable injection; context aware Twig global variables; simplified authorization checks|
+|[TagsBundle](https://github.com/netgen/TagsBundle)|helps with taxonomy management and classification of content|
+|[NovaeZSEOBundle](https://github.com/Novactive/NovaeZSEOBundle)|simplifies SEO management|
+|[EzSystemsPrivacyCookieBundle](https://github.com/ezsystems/EzSystemsPrivacyCookieBundle)|adds privacy cookie banner into Symfony 2 application|
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: You can remove `Symfony 2 application`, it's self-explanatory when talking about Platform. (Also below)

File: docs/getting_started/bundles_starter_pack.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+|[ezflow-migration-toolkit](https://github.com/ezsystems/ezflow-migration-toolkit)|set of tools that helps migrate data from legacy eZ Flow extension to eZ Studio landing page management|
+|[ezplatformsearch](https://github.com/netgen/ezplatformsearch)|eZ Publish Legacy extension that integrates eZ Platform search capabilities into eZ Publish Legacy|
+|[ngsymfonytools](https://github.com/netgen/ngsymfonytools)|eZ Publish 4 extension that provides a way to include Twig templates, as well as running Symfony sub-requests, directly from the eZ Publish legacy templates|
+|[NetgenRichTextDataTypeBundle](https://github.com/netgen/NetgenRichTextDataTypeBundle)|allows eZ Publish Legacy to work with RichText (ezrichtext) field type available in eZ Platform|
+
+## Contributors
+
+|Bundle|Description|
+|------|-----------|
+|[RepositoryProfilerBundle](https://github.com/ezsystems/RepositoryProfilerBundle)| profiles eZ Platform API/SPI and setup scenarios to be able to continuously test to keep track of performance regressions of repository and underlying storage engines|
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: `... profiles Platform API/SPI and sets up scenarios...`

File: docs/getting_started/bundles_starter_pack.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+## Educational
+
+This bundles are not necessarily something you would install but there are useful for learning process:
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: `but they are useful`

File: docs/getting_started/bundles_starter_pack.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+|[ezplatform-demo](https://github.com/ezsystems/ezplatform-demo)|fork of "ezplatform" meta repository, contains code and dependencies for demo distribution of eZ Platform. Not recommended for a clean installation for new projects, but great for observation and learning(example site)|
+|[TweetFieldTypeBundle](https://github.com/ezsystems/TweetFieldTypeBundle)|bundle that is created in the [Field Type Tutorial](../tutorials/field_type/creating_a_tweet_field_type.md) (example field type)|
+|[ezplatform-drawio-fieldtype](https://github.com/ezsystems/ezplatform-drawio-fieldtype)| provides support for diagrams editing in eZ Platform via draw.io (example field type)|
+|[ezplatform-ui-2.0-introduction](https://github.com/ezsystems/ezplatform-ui-2.0-introduction)|open source professional CMS (Content Management System) developed by eZ Systems and eZ Community|
```
Comment (DominikaK) [inline docs/getting_started/bundles_starter_pack.md]: Description looks like a copy from the main Platform repo. Maybe replace it with something along the lines of `an example of eZ Platform extensibility in version 2`?

File: docs/getting_started/bundles_starter_pack.md (hunk 3 above)
Comment (andrerom) [reply docs/getting_started/bundles_starter_pack.md]: Also text is outdated, it also supports symfony 3

fixed ;)

File: docs/getting_started/bundles_starter_pack.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+For more information about eZ bundle structure checkout:
+
+- [Bundle section](../guide/bundles.md) in our Guide.
+- List of [external bundles](https://ezplatform.com/Bundles) compatible with eZ Platform.
+
+## Everyday Platform usage
+
+|Bundle|Description|
+|------|-----------|
+|[ezmigrationbundle](https://github.com/kaliop-uk/ezmigrationbundle)|helps to handle content upgrades/migrations|
```
Comment (andrerom) [inline docs/getting_started/bundles_starter_pack.md]: can skip the ´s´ on upgrade and migration here I think, it is plural, but you only do once at a time

Or wdyt @DominikaK ?

File: docs/getting_started/bundles_starter_pack.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+|[EzCoreExtraBundle](https://github.com/lolautruche/EzCoreExtraBundle)|adds: configurable template variable injection; context aware Twig global variables; simplified authorization checks|
+|[TagsBundle](https://github.com/netgen/TagsBundle)|helps with taxonomy management and classification of content|
+|[NovaeZSEOBundle](https://github.com/Novactive/NovaeZSEOBundle)|simplifies SEO management|
+|[EzSystemsPrivacyCookieBundle](https://github.com/ezsystems/EzSystemsPrivacyCookieBundle)|adds privacy cookie banner into Symfony 2 application|
+|[CommentsBundle](https://github.com/ezsystems/CommentsBundle)|integrates comments with Disqus and Facebook allowing custom integrations|
```
Comment (andrerom) [inline docs/getting_started/bundles_starter_pack.md]: maybe `, also allows custom comment system/services integrations`

**Pull request merged**